AITK v0-16-1 docs update

ms-johnalex opened the pull request.

aprilgittens commented on docs/intelligentapps/agentbuilder.md:
```diff
@@ -82,21 +83,21 @@ To use an existing MCP server, follow these steps:
    1. Navigate back to **Tools** section and select **+ MCP Server**
-   2. Select the server you just configured from the dropdown list
+   2. Select the server you configured from the dropdown list
    3. Select tools you want to use.
```
@ms-johnalex this instruction is duplicated. It appears again in step 8.

aprilgittens commented on docs/intelligentapps/models.md:
```diff
@@ -117,6 +122,23 @@ To add a self-hosted or locally running Ollama model:
+
+You can deploy a model to Azure AI Foundry directly from the AI Toolkit. This allows you to run the model in the cloud and access it via an endpoint.
+
+1. From the model catalog, select the model you want to deploy
```
@ms-johnalex missing a period after 'deploy'

aprilgittens commented on docs/intelligentapps/agentbuilder.md:
```diff
@@ -82,21 +83,21 @@ To use an existing MCP server, follow these steps:
 4. Select tools from the MCP server if there are multiple tools available.
```
@ms-johnalex how come these numbers are '1, 2, 3, 4, 5, etc.' and not '1, 1, 1, 1, 1, etc.'?

ms-johnalex replied:
They should be 1,1,1....

aprilgittens reviewed (commented):
Hey @ms-johnalex

I provided some minor feedback.

MuyangAmigo reviewed (approved):
@ms-johnalex - The update looks great!

MuyangAmigo commented:
Hey @ntrogh - @ms-johnalex is now helping us with the AI Toolkit content, can you review the updates for v0.16.1 version of AITK doc? This is should be able to merge any time as AITK has already published.

ntrogh commented on docs/intelligentapps/models.md:
```diff
@@ -117,6 +122,23 @@ To add a self-hosted or locally running Ollama model:
+
+    ![AI Toolkit interface showing the model catalog with a model selected and the Deploy to Azure AI Foundry button highlighted.](./images/models/catalog-deploy-dropdown.png)
+
+1. In the **model deployment** tab, Enter the required information, such as the model name, description, and any additional settings, as in the following screenshot:
```
```suggestion
1. In the **model deployment** tab, enter the required information, such as the model name, description, and any additional settings, as in the following screenshot:
```

ntrogh commented on docs/intelligentapps/models.md:
```diff
@@ -38,16 +41,18 @@ To find a model in the model catalog:
 
         > [!TIP]
-        > You can edit the API key later by right clicking the model and selecting **Edit** and view the encrypted value in `${HOME}/.aikt/models/my-models/yml` file.
+        > You can edit the API key later by right     clicking the model and selecting **Edit** and view the encrypted value in `${HOME}/.aikt/models/my-models/yml` file.
```
```suggestion
        > You can edit the API key later by right-clicking the model and selecting **Edit** and view the encrypted value in `${HOME}/.aikt/models/my-models/yml` file.
```

ntrogh commented on docs/intelligentapps/models.md:
```diff
@@ -117,6 +122,23 @@ To add a self-hosted or locally running Ollama model:
+
+1. In the **model deployment** tab, Enter the required information, such as the model name, description, and any additional settings, as in the following screenshot:
+
+    ![AI Toolkit interface showing the model deployment tab with fields for model name, description, and additional settings.](./images/models/deploy-to-azure-dialog.png)
```
```suggestion
    ![Screenshot of the AI Toolkit interface showing the model deployment tab with fields for model name, description, and additional settings.](./images/models/deploy-to-azure-dialog.png)
```

ntrogh commented on docs/intelligentapps/playground.md:
```diff
@@ -23,7 +23,20 @@ To test a model in the playground, follow these steps:
+
+- When you reach GitHub’s model usage limit, AI Toolkit shows a warning with a link to GitHub’s paid usage documentation.
+
+    ![GitHub Pay-as-you-go Model Support](./images/playground/github_pay_as_you_go.png)
```
```suggestion
    ![Screenshot of the GitHub Pay-as-you-go Model support.](./images/playground/github_pay_as_you_go.png)
```

ntrogh commented on docs/intelligentapps/playground.md:
```diff
@@ -23,7 +23,20 @@ To test a model in the playground, follow these steps:
+
+    ![GitHub Pay-as-you-go Model Support](./images/playground/github_pay_as_you_go.png)
+
+- Turn on billing for models in your [GitHub Settings](https://github.com/settings/billing).
```
```suggestion
- Turn on billing for models in your [GitHub settings](https://github.com/settings/billing).
```

ntrogh reviewed (changes requested):
@ms-johnalex Added a few minor suggestions.

ms-johnalex commented:
@ntrogh - In addition to your requested changes, I updated all used image files from underscores to hyphens, changed image links accordingly, and added updated alt-text.

ntrogh reviewed (approved):
@ms-johnalex LGTM
